source/triggers/database-triggers.txt

```diff
@@ -331,10 +297,37 @@ following table describes these options.
    You can upgrade a non-sharded cluster (with preimages) to a 
    sharded cluster, as long as the cluster is running 5.3 or later.
 
-Function
+Trigger Configurations
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :header-rows: 1
+   :widths: 15 30
+
+   * - Field
+     - Description
+
+   * - :guilabel:`Auto-Resume Triggers`
+
+     - .. include:: /includes/trigger-auto-resume.rst
+
+   * - :guilabel:`Event Ordering`
+     
+     - If enabled, trigger events are processed in the order in which they occur. 
+       If disabled, events can be processed in parallel, which is faster when 
+       many events occur at the same time.
+
+       .. include:: /includes/trigger-event-ordering.rst
```

Comment:
I know you didn't add this, but one of our current org objectives is to reduce nested components. This include has an admonition component, which means we have a table with and admonition.

It would be good to fix this. In this instance, it's OK to copy-paste the admonition content into the table cell.

Reply:
Oof. Actually, I see we have a _lot_ of nested components like that on this page. Feel free to ignore them for this PR.